src/engine/qcommon/files.cpp

```diff
@@ -31,9 +31,19 @@ along with Daemon Source Code.  If not, see <http://www.gnu.org/licenses/>.
 const char TEMP_SUFFIX[] = ".tmp";
 
 // Cvars to select the base and extra packages to use
-static Cvar::Cvar<std::string> fs_basepak("fs_basepak", "base pak to load", 0, DEFAULT_BASE_PAK);
+static Cvar::Cvar<std::string>* fs_basepak;
```

Comment:
Why does this need to be a pointer? Can't you just change its value instead of allocating a new one?